### ISSUE TITLE
Missing some JARs

### ISSUE THREAD

**Issue by sethk** [databrary/datavyu#357]
**Describe the bug**
The following JAR files are not available from any Maven repository:
quaqua-filechooser-only-7.0.1.jar
quaqua-osx64-7.0.1.jar
libNativeOSXCanvas-0.92.jar

Some of these may be available from the datavyu Maven repository, but it yields HTTP 404:
http://datavyu.org/maven2

**To Reproduce**
Try to build dev branch using IntelliJ IDEA 2020.1, Maven 3.6.1.

**Expected behavior**
Should be able to fetch all the dependencies.

**Desktop (please complete the following information):**
macOS 10.15.5

**Additional context**
To work around this I installed the JARs from the 1.5.1:
`sh ~/Library/Application\ Support/JetBrains/Toolbox/apps/IDEA-U/ch-0/201.6668.121/IntelliJ\ IDEA.app/Contents/plugins/maven/lib/maven3/bin/mvn install:install-file -Dfile=/Applications/datavyu\ 1.5.1.app/Contents/Java/lib/quaqua-osx64-7.0.1.jar -DgroupId=ch.randelshofer.quaqua -DartifactId=quaqua-osx64 -Dversion=7.0.1 -Dpackaging=jar`
etc.

**Comment by TheoWolf** [comment]
@sethk Indeed, the dependencies are missing due to a recent domain change, these dependencies are hosted on the datavyu server. I will add them to Bintray so they will be accessible to contributors
